core/src/test/scala/org/http4s/rho/RhoRoutesSuite.scala

```diff
@@ -63,7 +63,9 @@ class RhoRoutesSuite extends CatsEffectSuite with RequestRunner {
         Ok("twoparams2_" + foo + bar.getOrElse("cat"))
     }
 
-    GET / "variadic" / * |>> { tail: List[String] => Ok("route8_" + tail.mkString("/")) }
+    //identity is used to force the type usage of the path tail
+    GET / "variadic" / * |>> { tail: List[Segment] =>
+      Ok("route8_" + tail.map(identity).mkString("/")) }
```

Comment:
```suggestion
    //encoded is used to force the type usage of the path tail
    GET / "variadic" / * |>> { tail: List[Segment] =>
      Ok("route8_" + tail.map(_.encoded).mkString("/")) }
```

Reply:
@vaslabs are you happy with incorporating @zarthross ' suggestion?

Reply:
I thought I pushed it already but maybe I only did on the other branch

Reply:
done